lite/kernels/xpu/instance_norm_compute.cc

```diff
@@ -25,12 +25,32 @@ void InstanceNormCompute::Run() {
   auto& param = this->Param<param_t>();
   auto& ctx = this->ctx_->As<XPUContext>();
   auto x_dims = param.x->dims();
-  CHECK_EQ(x_dims.size(), 4);
+  bool x_dims_support = (x_dims.size() == 4 || x_dims.size() == 5);
+  CHECK_EQ(x_dims_support, true);
```

Comment:
CHECK(x_dims.size() == 4 || x_dims.size() == 5)<<"Not support x_dims_rank = "<<x_dims.size();